Web_Development/FrontEnd/HTML_Lists/Readme.md

```diff
@@ -0,0 +1,514 @@
+# HTML LISTS_#1055
```

Comment:
You don't need to have that number there. But first correct your content ( without plagiarism )

Reply:
okay I'll change it ... didn't knew as its my first time of creating  a PR.